templates/promoted.html

```diff
@@ -0,0 +1,40 @@
+{% extends "_layout.html" %}
+
+{% block title %}
+  Promoted Snaps — Linux software in the Snap Store
```

Comment:
I'm not sure - is 'promoted snaps' official way how they are called? Is that the title we want for that page?